### PR TITLE
docs: add 2.13 Release Notes

### PR DESCRIPTION
**Pull request by yzeng25**
Changes:

Add 2.13 Release Notes.

### CODE REVIEW

File: website/blog/2022/03/28/release-apache-apisix-2.13.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    title: "Technical Writer"
+    url: "https://github.com/yzeng25"
+    image_url: "https://github.com/yzeng25.png"
+keywords: 
+- Apache APISIX
+- 2.13.0
+- Release Notes
+- API Gateway
+- LTS version
+description: Today, the Apache APISIX community is bringing a new LTS release, 2.13.0, which not only provides more stable performance, but also supports more observability and service discovery plugins, and a better multilingual development system.
```
Comment (SylviaBABY) [inline website/blog/2022/03/28/release-apache-apisix-2.13.md]:
```suggestion
description: Apache APISIX community is bringing a new LTS release, 2.13.0, which not only provides more stable performance, but also supports more observability and service discovery plugins, and a better multilingual development system.
```

File: website/blog/2022/03/28/release-apache-apisix-2.13.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+tags: [Release]
+---
+
+> Today, the Apache APISIX community is bringing a new LTS release, 2.13.0, which not only provides more stable performance, but also supports more observability and service discovery plugins, and a better multilingual development system.
```
Comment (SylviaBABY) [inline website/blog/2022/03/28/release-apache-apisix-2.13.md]:
```suggestion
> Apache APISIX community is bringing a new LTS release, 2.13.0, which not only provides more stable performance, but also supports more observability and service discovery plugins, and a better multilingual development system.
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    title: "Technical Writer"
+    url: "https://github.com/yzeng25"
+    image_url: "https://github.com/yzeng25.png"
+keywords: 
+- Apache APISIX
+- 2.13.0
+- 版本发布
+- API 网关
+- LTS version
+description: 今天，Apache APISIX 社区带来了一个全新的 LTS 版本——2.13.0。该 LTS 版本不仅性能更加稳定，而且支持了更多的可观测性、服务发现插件和更完善的多语言开发体系。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md]:
```suggestion
description: 全新的 LTS 版本—— Apache APISIX 2.13.0 正式发布。该 LTS 版本不仅性能更加稳定，而且支持了更多的可观测性、服务发现插件和更完善的多语言开发体系。
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+tags: [Release]
+---
+
+> 今天，Apache APISIX 社区带来了一个全新的 LTS 版本——2.13.0。该 LTS 版本不仅性能更加稳定，而且支持了更多的可观测性、服务发现插件和更完善的多语言开发体系。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md]:
```suggestion
> 全新的 LTS 版本—— Apache APISIX 2.13.0 正式发布。该 LTS 版本不仅性能更加稳定，而且支持了更多的可观测性、服务发现插件和更完善的多语言开发体系。
```

File: website/blog/2022/03/28/release-apache-apisix-2.13.md (hunk 2 above)
Comment (EstelleRao) [reply website/blog/2022/03/28/release-apache-apisix-2.13.md]: Apache APISIX community has rolled out a new LTS version, 2.13.0. This version not only delivers more stable performance, but also supports more observability and service discovery plugins, as well as a more robust multilingual development system.

**Review by SylviaBABY — DISMISSED** [review]

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+当然，2.13.0 版本还有其他的新变化，比如修复了历史版本的不合理行为。如需了解具体优化信息，请查阅 [2.13.0 Changelog](https://github.com/apache/apisix/blob/release/2.13/docs/zh/latest/CHANGELOG.md#2130)。
+
+### 新功能：可观测性层面对接更多的监控体系
+
+作为 API 网关，Apache APISIX 一直致力于连接更多的服务，打通更多的可观测性上下游。我们在每个版本都会为此添砖加瓦，2.13.0 版本也不例外。
+
+**这次我们新增了一个 tracing 插件：`opentelemetry`，允许发送 OpenTelemetry tracing 数据到配置的 collector。**下面简单通过一个示例来看一下。
+
+在静态配置里面设置了 collector：
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md:51]:
```suggestion
在静态配置里面设置 collector：
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+      max_queue_size: 6
+      batch_timeout: 2
+      inactive_timeout: 1
+      max_export_batch_size: 2
+```
+
+之后就可以在特定的路由上开启 tracing：
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md]:
```suggestion
curl http://127.0.0.1:9080/apisix/admin/routes/1  \
-H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+```
+
+命中该路由的请求将会上报 OpenTelemetry 的数据到对应的 collector。
+
+此外，我们还新增了两个日志插件，支持把日志上报到 ClickHouse 和 Loggly 中。
+
+ClickHouse 是地表最快的 OLAP 数据库之一。Apache APISIX 支持发送 access log 和 error log 到 ClickHouse，示例如下：
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md]: ditto

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+           "nodes": {
+               "127.0.0.1:1980": 1
+           }
+      },
+      "uri": "/hello"
+}'
+```
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/plugin_metadata/error-log-logger -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md]: ditto

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+  }
+}'
+```
+
+Loggly 是 SolarWinds 旗下的日志处理 SaaS 平台，我们支持通过 syslog 或 HTTP/HTTPS 的方式发送 access log。示例如下：
+
+配置上报方式
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/plugin_metadata/loggly -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md]: ditto

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+{
+   "protocol": "http"
+}'
+```
+
+配置需要上报的路由
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/03/28/release-apache-apisix-2.13.md]: diito

File: website/blog/2022/03/28/release-apache-apisix-2.13.md [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+      max_queue_size: 6
+      batch_timeout: 2
+      inactive_timeout: 1
+      max_export_batch_size: 2
+```
+
+After that, tracing can be enabled on a specific route.
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (guitu168) [inline website/blog/2022/03/28/release-apache-apisix-2.13.md]: Refer to the Chinese

**Review by SylviaBABY — APPROVED** [review]

**Review by guitu168 — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**